build.sbt

```diff
@@ -163,7 +163,11 @@ GatherLicenses.distributions := Seq(
   makeStdLibDistribution("Database", Distribution.sbtProjects(`std-database`)),
   makeStdLibDistribution("Image", Distribution.sbtProjects(`std-image`)),
   makeStdLibDistribution("AWS", Distribution.sbtProjects(`std-aws`)),
-  makeStdLibDistribution("Snowflake", Distribution.sbtProjects(`std-snowflake`))
+  makeStdLibDistribution(
+    "Snowflake",
+    Distribution.sbtProjects(`std-snowflake`)
+  ),
+  makeStdLibDistribution("Microsoft", Distribution.sbtProjects(`std-microsoft`))
```

Comment:
I assume this was discussed, so not forcing any changes, but it feels to me that a package called `Microsoft` may be too broad - they have **so many** different products that could fit into this library - I can't imagine as we progress and add more integrations with other MS products that they'd fit into a single library.

Was it considered to call this `MS_SQL_Server` or something like that?

Reply:
I think I started with SQLServer and @jdunkerley convinced me it should be Microsoft

Reply:
We have AWS and Google. Having the full set of the big three feels correct, so Standard.Microsoft. We considered moving Excel but it is so core its almost like plain text must be support by default.

Yes I expect all of them will end up with sub-libraries that you include what you need one day.